src/engine/controls/bpmcontrol.cpp

```diff
@@ -800,6 +802,10 @@ double BpmControl::getBeatMatchPosition(
 
     double dOtherPosition = pOtherEngineBuffer->getExactPlayPos();
 
+    double dOtherPrevBeat;
+    double dOtherNextBeat;
+    double dOtherBeatLength;
+    double dOtherBeatFraction;
```

Comment:
for safety, can we initialize these to some useful value?  0 or -1 or whatever

Reply:
Oh Yes, this reveals also a bug that we use -1 below. Now I have cached that case as well.